examples/functional_tests.py

```diff
@@ -0,0 +1,94 @@
+import sys
+from openrazer.client import DeviceManager
+from openrazer.client import constants as razer_constants
+
+def main():
+    # Create a DeviceManager. This is used to get specific devices
+    device_manager = DeviceManager()
+
+
+    print("Found {} Razer devices".format(len(device_manager.devices)))
+    print()
+
+
+    prompt = 'noprompt' not in sys.argv
+
+    # Iterate over each device and pretty out some standard information about each
+    for device in device_manager.devices:
+        for capability, supported in device.capabilities.items():
+            if not supported:
+                continue
+
+            if capability == "name":
+                print(f"device.name: {device.name}")
+            elif capability == "type":
+                print(f"device.type: {device.type}")
+            elif capability == "firmware_version":
+                print(f"device.firmware_version: {device.firmware_version}")
+            elif capability == "serial":
+                print(f"device.serial: {device.serial}")
```

Comment:
Putting all of these bit of info into a `get_basic_info` function could be helpful for showing information that is expected. I think if any of these are missing then that should be a red flag that something isn't working quite right. One could imaging an output like the following.
```
Name:  Death Adder Gen X
Device Type: Mouse
Firmware Version: MISSING!
Serial Number: MISSING!
```
Not sure if that sort of info is outside the scope. If so, ignore me. :smile: